test/unit/spec/MfaVerify_spec.js

```diff
@@ -4144,6 +4154,152 @@ function (Okta,
         });
       });
 
+      Expect.describe('Multiple Security Keys (U2F) are setup with Security Question', function () {
```

Comment:
should we name this `Multiple Security Keys (U2F) and one more factor are setup` since we don't really care if it is security question or something else?

Reply:
👍